iDiff/utils/tar_utils.go

```diff
@@ -40,6 +27,21 @@ func UnTar(filename string, path string) error {
 		}
 		if err != nil {
 			glog.Fatalf(err.Error())
+			return err
+		}
+
+		if strings.Contains(header.Name, ".wh.") || strings.Contains(header.Name, "/.wh.") {
```

Comment:
if `header.Name` contains `/.wh.`, wouldn't it always contain `.wh.` as well? am I missing something?